apps/web/.eslintrc.cjs

```diff
@@ -6,7 +6,10 @@ module.exports = {
 			rules: {
 				// allow devdependencies
 				// https://github.com/weiran-zsd/eslint-plugin-node/blob/cb4f099ae928d627abfc635966622cf4266a7b1c/docs/rules/no-unpublished-import.md
-				'n/no-unpublished-import': 'off'
+				'n/no-unpublished-import': 'off',
+				'jest/valid-expect': 'off',
+				'no-unused-vars': 'off'
```

Comment:
no unused vars could be useful to clean up the code actually

Reply:
fair game - put it back, although was only for tests and infrastructure code